src/zenml/integrations/pytorch_lightning/__init__.py

```diff
@@ -21,7 +21,7 @@ class PytorchLightningIntegration(Integration):
     """Definition of PyTorch Lightning integration for ZenML."""
 
     NAME = PYTORCH_L
-    REQUIREMENTS = ["pytorch_lightning<2.0.0"]
+    REQUIREMENTS = ["pytorch_lightning"]
```

Comment:
Does this mean that our integration works for both pre-2.0 as well as post-2.0 releases?